mautic/mautic
# Plugin form loading

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
|Enhancement | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR changes field matching to pagination
[//]: # ( As applicable: )

#### Steps to test this PR:
1.  apply this PR and see fields listed through pages
2. test that fields are mapped correctly
